### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by peschee**
Reference `mwc` branch instead of `master`.

### CODE REVIEW

**Comment by maryla-uc** [comment]
Could someone merge this please? All links are currently broken which is very confusing.

File: README.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ Material Web uses modern browser features that are natively supported in the lat
 
 #### Web Components
 
-To support Web Components in IE11 and other older browsers, install the [Web Components Polyfills](https://github.com/webcomponents/polyfills/tree/master/packages/webcomponentsjs):
+To support Web Components in IE11 and other older browsers, install the [Web Components Polyfills](https://github.com/webcomponents/polyfills/tree/mwc/packages/webcomponentsjs):
```
Comment (maryla-uc) [inline README.md:201]: I think this one shouldn't be changed

Comment (peschee) [reply README.md]: I have adapter that path

**Comment by peschee** [comment]
Any chance someone can take a look at this and merge?

**Pull request merged**